flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/AbstractPrometheusReporter.java

```diff
@@ -321,16 +326,23 @@ static class HistogramSummaryProxy extends Collector {
 
         private final Map<List<String>, Histogram> histogramsByLabelValues = new HashMap<>();
 
+        private final Boolean histogramMaxEnabled;
+        private final Boolean histogramMinEnabled;
```

Comment:
I think we should make the entire `QUANTILES` a configurable list, rather than introducing min/max as two special cases.